docs/choreolib/auto-factory.md

```diff
@@ -309,3 +309,43 @@ public class Robot extends TimedRobot {
     }
 }
 ```
+
+## AutoChooser for AdvantageKit Users
+Choreo's base `AutoChooser` does not support log replay. AdvantageKit users should instead copy-and-paste the code below, and use the `AutoChooserAK` class instead of the base `AutoChooser` class.
+
+!!! tip
+    `AutoChooserAK` will automatically register the auto chooser to smart dashboard, meaning that
+    a manual `SmartDashboard.putData` call is no longer necessary.
+
+```java title="AutoChooserAK.java"
+package your.utilities.folder;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.Logger;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+public class AutoChooserAK extends AutoChooserImpl {
```

Comment:
Is AutoChooserImpl the right thing to extend here? Also this example needs to import AutoChooser, no?